tests/components/plaato/test_config_flow.py

```diff
@@ -237,50 +234,75 @@ async def test_show_config_form_api_method_no_auth_token(hass, webhook_id):
 
 async def test_options(hass):
     """Test updating options."""
-    config_entry = MockConfigEntry(
-        domain=DOMAIN,
-        title="NAME",
-        data={},
-        options={CONF_SCAN_INTERVAL: 5},
-    )
+    with patch(
+        "homeassistant.components.plaato.async_setup", return_value=True
+    ) as mock_setup, patch(
+        "homeassistant.components.plaato.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        config_entry = MockConfigEntry(
```

Comment:
The config entry can be created and added to hass outside the patch context manager. It's just the config entry set up and block till done that needs to go inside the context manager.

Reply:
OK. took inspiration from a different component. Will fix it.